Common/UnitDefinitions/Compressibility.json

```diff
@@ -0,0 +1,29 @@
+{
+  "Name": "Compressibility",
+  "BaseUnit": "InversePascal",
+  "XmlDoc": "Compressibility is the measure of the relative volume change of a fluid or solid in response to pressure changes.",
+  "BaseDimensions": {
+    "L": 1,
+    "M": -1,
+    "T": 2
+  },
+  "Units": [
+    {
+      "SingularName": "InversePascal",
+      "PluralName": "InversePascals",
```

Comment:
Would any other units be useful to add also @sveipa? I don't know this domain.

https://petroleumoffice.com/unitconverter/compressibility/
https://www.unitsconverters.com/en/Compressibility-Conversions/Measurement-1170
https://www.calculand.com/unit-converter/?gruppe=Compressibility&einheit=Second+per+poise+%5Bs%2FP%5D

Reply:
We did a scan of our legacy code here and these are the important units + scales for compressibilty:
Pa-1 - 1/Pascal

kPa-1 - 1/kiloPascal

MPa-1 - 1/MegaPascal

atm-1 - 1/atmosphere

mbar-1 - 1/millibar

bar-1 - 1/bar

psi-1 - 1/pounds per square inch